product_import/wizard/product_import.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2022 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+from base64 import b64decode, b64encode
+from datetime import date, timedelta
+
+from lxml import etree
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+logger = logging.getLogger(__name__)
+
+
+class ProductImport(models.TransientModel):
+    _name = "product.import"
+    _description = "Product import from files"
+
+    product_file = fields.Binary(
+        string="Product Catalogue",
```

Comment:
```suggestion
        string="Product Catalogue",
        attachment=True,
```
AFAIR this is not the default in v14

Reply:
It is default in Odoo 14